lib/encoder.py

```diff
@@ -836,29 +857,33 @@ def StoreEncoding(self, encoding, workdir=None):
   def ReadEncodingResult(self, encoding, scoredir=None):
     """Reads an encoding result back from storage, if present.
 
-    None is returned file does not exist."""
+    None is returned if file does not exist.
+    If scoredir is given, only that directory is searched.
```

Comment:
Yeah, I don't mean you should add it everywhere. Such documentation is probably just waste and maintenance overhead in most cases, unless you have a public API somewhere that needs good documentation. What I meant is that the docstring above already mostly has the information of a full proper docstring, and then you may as well write it in that format. That's just a mild preference though.